docs/cudf/source/index.rst

```diff
@@ -8,6 +8,7 @@ Welcome to cuDF's documentation!
    api.rst
    10min.ipynb
    basics.rst
+   io-supported-types.rst
```

Comment:
I think It would be nice to make an `Input / Output` section generally. I can imagine the next step for docs will be documenting which readers and variants are GPU-accelerated versus falling back to CPU and whatnot.

Reply:
Made these changes and created a separate section for I/O stuff.